docker-compose.yaml

```diff
@@ -1,4 +1,4 @@
-version: "3.9"
+version: "3.3"
```

Comment:
Dlaczego?

Reply:
Jak mam wersję 3.9 to wyskakują mi takie błędy:

ERROR: Version in "./docker-compose.yaml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/
ERROR: Version in "./docker-compose.yaml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/
ERROR: Version in "./docker-compose.yaml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/

Dla wersji 3.3 działa mi normalnie

Reply:
To jest wersja chematu docker-compose.yml, a nie Dockera, @Gumkle ;) a 3.9 jeszcze nie ma.

https://docs.docker.com/compose/compose-file/compose-file-v3/

Reply:
@Helga151 spróbuj zrobić sobie update compose, być może masz jakąś starszą wersję. @marcinlawnik spoko tylko to w niczym nie przeszkadza, bo działa normalnie, a na tej samej stronie którą linkujesz w examplach często  jest odwołanie do wersji 3.9. Możemy to zmienić na 3.8 dla porządku, ale wolałbym tego nie zmieniać aż do 3.3

Reply:
@Gumkle wersja zmieniona i compose updatowany